pages/index.vue

```diff
@@ -1,8 +1,72 @@
 <template>
-  <div class="flex h-screen flex-col items-center justify-center font-bold">
-    <span class="bg-cyan-100 text-xl"
-      >This is the Modernized Name-Examination.</span
-    >
-    <span class="bg-gray-100 text-lg">Under Construction</span>
+
+<div class="w-10/12">
+
+  <div class="text-2xl text-gray-800 font-semibold relative top-28 left-72" v-if="!this.auth">
+    <span>Your authorization is missing or has expired. Please login.</span>
   </div>
+
+  <div class="text-2xl text-gray-800 relative top-10 left-72" v-else>
+    <header class="font-bold text-5xl mb-10">Welcome to Name X!</header>
+    <span>Canada's most modern, semi-automated way to examine business name requests. Automated indicators show you:</span>
+
+    <div class="m-10">
+      <div class=" my-10">
+        <header class = "text-3xl font-semibold">Conflicts</header>
+        <span class="mt-6">Searches the name against related industry categories, synonyms, word substitutions (eg. 9 and nine), and more. The conflict search is only as good as the information in it, so if you don't see something come back in the search you should check the SOLR admin tool to see if it is in the correct industry category or other search options.</span>
+      </div>
+
+      <div class="my-10">
+        <header class = "text-3xl font-semibold">Condition</header>
+        <span class="mt-6">This gives you the messages to clients and internal information based on condition of words or phrases, eg. 'Doctor', 'BC', and such.</span>
+      </div>
+
+      <div class="my-10">
+        <header class = "text-3xl font-semibold">Trademarks</header>
```

Comment:
ESLint should fix this to `class="...`.

I recommend you add some ESLint plugins, some of which auto-fix your code in the same way prettier does. Example:
https://github.com/bcgov/business-filings-ui/blob/79313d1bbec714b8f18c2d34d8111b9e831b4211/.eslintrc.js#L6